tests/test_cli.py

```diff
@@ -311,6 +324,17 @@ def test_show_gpus(self, monkeypatch):
                                        ['V100:4', '--cloud', cloud, '--all'])
             assert isinstance(result.exception, SystemExit)
 
+    def test_show_gpus_non_gpu_accelerator(self, monkeypatch):
+        """Trainium should be returned even though it is not a GPU."""
+        mock_api_server_calls(monkeypatch)
+
+        cli_runner = cli_testing.CliRunner()
+        result = cli_runner.invoke(command.show_gpus, ['Trainium'])
+
+        assert result.exit_code == 0
+        assert 'Trainium' in result.stdout
```

Comment:
What about we check the instance type in the result, e.g. `trn1n.32xlarge` and `trn2.48xlarge` are both in the result?

Reply:
Line 335 can be removed as it will exist even if the chip is not in the result.